scripts/fix_hg38.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
```

Comment:
Some comments describing what this does would be helpful.

Reply:
I removed the `scripts` folder. These are left over from processing Fasta files for a local higlass app.